webapp/src/main/webapp/news-extensions/extensions.js

```diff
@@ -95,7 +95,7 @@ export function initExtensions() {
   if (eXo.env.portal.spaceId) {
     canUserCreateNews(eXo.env.portal.spaceId)
       .then(canCreateNews => {
-        switchToArticleActivityComposerPlugin.enabled = newsActivityComposerPlugin.enabled = eXo.env.portal.spaceId && canCreateNews;
+        switchToArticleActivityComposerPlugin.enabled = newsActivityComposerPlugin.enabled = (eXo.env.portal.spaceId && eXo.env.portal.spaceId !== '') && canCreateNews;
```

Comment:
I am not sure that this check is useful
`eXo.env.portal.spaceId !== ''`
check https://developer.mozilla.org/en-US/docs/Glossary/Falsy

Reply:
> I am not sure that this check is useful
> `eXo.env.portal.spaceId !== ''`
> check https://developer.mozilla.org/en-US/docs/Glossary/Falsy

I think it's necessary to check if `eXo.env.portal.spaceId !== ''`, The main stream has an empty value on `eXo.env.portal.spaceId  so we should do this check.